apps/libs/filters/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './ErrorFilter'
+export * from './GlobalValidationPipe'
+export * from './ResponseInterceptor'
```

Comment:
Maybe should split this into another PR too. I think would be better to keep this atomic.

Reply:
resolved as #1149